Directory.Build.targets

```diff
@@ -129,7 +129,7 @@ or
     <Error Condition="!Exists('$(SentryAndroidRoot)')" Text="Couldn't find the Android root at $(SentryAndroidRoot)."></Error>
     <Message Importance="High" Text="Building Sentry Android SDK."></Message>
 
-    <Exec WorkingDirectory="$(SentryAndroidRoot)" Command="./gradlew :sentry-android-core:assembleRelease :sentry-android-ndk:assembleRelease :sentry:jar --no-daemon --stacktrace --warning-mode all"></Exec>
+    <Exec WorkingDirectory="$(SentryAndroidRoot)" Command="./gradlew :sentry-android-core:assembleRelease :sentry-android-ndk:assembleRelease :sentry:jar --no-daemon --stacktrace --warning-mode none"></Exec>
```

Comment:
Sure that's case? As far as I recall `core` and `ndk` don't depend on each other. At runtime we try to load NDK via reflection: https://github.com/getsentry/sentry-java/blob/61ea65371ae9c7ea52ecefcc6b67817f47f4c042/sentry-android-core/src/main/java/io/sentry/android/core/NdkIntegration.java#L50

So the order between them wouldn't matter other than both depend on core `sentry` package, but any build tool must resolve dependencies and build those first so might be even fine to remove `:sentry:jar` from it. Didn't try that out though.

Reply:
I didn't know! Thanks for the explanation!

Reply:
`:sentry:jar` should be the first just for being semantically correct, since the other 2 libs depend on `sentry`, it'd not even be needed to add here tho (transitive dependencies are compiled automatically anyway).
`sentry-android-core` and `sentry-android-ndk` don't depend on each other, so the order does not make any difference.